content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst

```diff
@@ -1,87 +1,76 @@
-products require no particular configuration.
-
-Once the products are configured, create a *bill of materials* for the kit product. Add each component
-and its quantity. Select the *BoM Type* Kit. All other options may be left with their default values.
-
-.. image:: kit_shipping/kit-bom.png
-    :align: center
-
-Manage Stock of Kit Product and Component Products
-==================================================
-
-If you want to manage the stock of the *top-level kit product*, you will use a standard *BoM Type* 
-manufacture or subcontracting instead. In this case you will either buy the final product from your
-subcontractor, or make it yourself through a manufacturing order. 
-
-In that case the product type will be storable, and the BOM type either 'Manufacture this product' 
-or 'Subcontracting' as shown below.
-
-.. image:: kit_shipping/kit-storable.png
-    :align: center
-
-
-Build a cleaner BOM
-==================== 
-
-As suggested above, a kit BOM can also simply be used to manage a more complex BOM. 
-
-Imagine the Custom Computer parts above were part of another final product. It would be much 
-clearer and simpler if the BOMs were added together (as shown below) instead of adding all 
-the individual parts one by one.
-
-.. image:: kit_shipping/kit-bom4.png
-    :align: center
-
-This is even better illustrated when looking at the BOM Structure Report, where it is 
-easy to expand and hide specific kit level or sub-assembly type BOMs. 
-
-.. image:: kit_shipping/kit-bom5.png
-    :align: center
-
-On the creation of the manufacturing order, the BOM will automatically expand to show all
-top level and kit level components. You can go back to the BOM at any point though as it 
-remains  available on the form view of your manufacturing order. 
-
-.. image:: kit_shipping/kit-bom6.png
-    :align: center
-
-Finally, note that if there were any Operations in the Kit BOM, these would also simply be added 
-in to the work order list of the main component. 
\ No newline at end of file
+========
+Use Kits
+========
+
+A *kit* is a set of unassembled components. Kits may be sold as products, but are also useful tools
+for managing more complex bills of materials (BoMs). To use kits, the :guilabel:`Manufacturing` and
+:guilabel:`Inventory` applications need to be installed.
+
+Create the kit as a product
+===========================
+
+To use a kit as a sellable product or simply as a component organization tool, the kit should first
+be created as a product. To create a kit product, go to :menuselection:`Manufacturing or Inventory
+--> Products --> Products`, and then click :guilabel:`Create`.
+
+Next, set the :guilabel:`Product Type` to :guilabel:`Storable Product`: this allows for management
+of kit inventory. The :guilabel:`Route` designation under the :guilabel:`Inventory` tab does not
+matter, since Odoo uses the routes of the kit’s components for replenishment purposes. All other
```

Comment:
replace serif apostrophe (which was likely copied over from a gdoc) with sans-serif version so it's more RST-friendly.
```suggestion
matter, since Odoo uses the routes of the kit's components for replenishment purposes. All other
```